cron/disabled/reject-contributions.js

```diff
@@ -78,7 +79,7 @@ async function run({ dryRun = false } = {}) {
       if (!transaction.RefundTransactionId) {
         if (!dryRun) {
           logger.info(`  - Refunding transaction`);
-          await libPayments.refundTransaction(transaction);
+          await libPayments.refundTransaction(transaction, fromCollectiveUser);
```

Comment:
@sbinlondon How were you able to catch this error?

Reply:
`refundTransaction` doesn't execute if you don't include the user. It does the refund and then calls `createRefundTransaction` which requires the `user` argument to be filled in.

This is why I wanted input on @Betree's suggestion to make `user` nullable https://github.com/opencollective/opencollective-api/pull/4712#discussion_r505210882

Reply:
Good catch! Let's make `user` nullable.